vllm/utils.py

```diff
@@ -467,3 +467,52 @@ def merge_dicts(dict1: dict[Any, list[Any]],
         merged_dict[key].extend(value)
 
     return dict(merged_dict)
+
+# Copyright 2023 The vLLM team.
+# Adapted from
+# https://github.com/NVIDIA/Megatron-LM/blob/main/megatron/core/tensor_parallel/utils.py
+# Copyright (c) 2022, NVIDIA CORPORATION. All rights reserved.
+from typing import Sequence
+
+import torch
```

Comment:
move the import to the top?

Reply:
Or move the utils to `vllm/distributed/utils.py`

Reply:
will do linter later.